monochart/templates/deployment.yaml

```diff
@@ -139,13 +139,14 @@ spec:
 {{- end }}
 {{- end }}
 
-        {{- if .Values.FirstVolumeMounts }}
         volumeMounts:
+        {{- if .Values.FirstVolumeMounts }}
```

Comment:
Please note that "volumeMounts:" will appear in the template but stay empty if no volumes are going to be mounted. Keeping it empty is compatible with helmfile/helm/kubectl and during my manual tests didn't introduce any issues. Example snippet:

```
        command:
          - sleep
        args:
          - 1d
        ports:
          - name: default
            containerPort: 5000
            protocol: TCP

        volumeMounts:
        resources:
          limits:
            cpu: 500m
            memory: 500Mi
          requests:
            cpu: 10m
            memory: 100Mi
```